src/app/views/login/login.controller.spec.js

```diff
@@ -42,12 +42,13 @@ describe('Login controller', () => {
     expect(ionicSideMenuDelegateStub.canDragContent.withArgs(true)).to.be.calledOnce;
   });
 
-  it('on submit should not try to login when form invalid', () => {
+  it('on submit should not try to login but show error when form invalid', () => {
```

Comment:
Lo mismo que comentamos el otro día con los nombres de los tests. Should not try...

Reply:
ok, cierto.